### PR TITLE
VR > Nav > Fix white spaces.

### PR DESCRIPTION
**Pull request by juanfra**
🎫 https://moderntribe.atlassian.net/browse/TEC-3131
🎥 https://www.loom.com/share/144613f48d07469e91d4d48998aa6930

### CODE REVIEW

File: src/views/v2/list/nav/next-disabled.php [hunk 1]
```diff
@@ -9,19 +9,20 @@
...
-				/* translators: %s: Event (plural or singular). */
-				sprintf( __( 'Next %1$s', 'the-events-calendar' ), $events_label ),
-				[ 'span' => [ 'class' => [] ] ]
-			);
-		?>
-	</button>
+	<button 
+		class="tribe-events-c-nav__next tribe-common-b2 tribe-common-b1--min-medium" 
+		disabled
+	><?php
```
Comment (tr1b0t) [inline src/views/v2/list/nav/next-disabled.php]: :no_entry_sign: Opening PHP tag must be on a line by itself

<sub>hash: 4101a1f40776af81a0c171c934e1e869</sub>

File: src/views/v2/list/nav/next-disabled.php [hunk 2]
```diff
@@ -9,19 +9,20 @@
...
+		$events_label = '<span class="tribe-events-c-nav__next-label-plural">' . tribe_get_event_label_plural() . '</span>';
+		echo wp_kses(
+			/* translators: %s: Event (plural or singular). */
+			sprintf( __( 'Next %1$s', 'the-events-calendar' ), $events_label ),
+			[ 'span' => [ 'class' => [] ] ]
```
Comment (tr1b0t) [inline src/views/v2/list/nav/next-disabled.php]: * :no_entry_sign: Short array syntax is not allowed
* :no_entry_sign: Short array syntax is not allowed
* :no_entry_sign: Short array syntax is not allowed

<sub>hash: ee4f53aa325bd162bb26825b48e72b20</sub>

File: src/views/v2/list/nav/next-disabled.php [hunk 3]
```diff
@@ -9,19 +9,20 @@
...
+		);
+	?></button>
```
Comment (tr1b0t) [inline src/views/v2/list/nav/next-disabled.php]: * :no_entry_sign: Line indented incorrectly; expected 2 tabs, found 1
* :no_entry_sign: Closing PHP tag must be on a line by itself

<sub>hash: db8c962eed62e90cffae0c78559e30ed</sub>

File: src/views/v2/list/nav/next.php [hunk 4]
```diff
@@ -21,14 +21,12 @@
 		rel="next"
 		class="tribe-events-c-nav__next tribe-common-b2 tribe-common-b1--min-medium"
 		data-js="tribe-events-view-link"
-	>
-		<?php
-			$events_label = '<span class="tribe-events-c-nav__next-label-plural"> ' . tribe_get_event_label_plural() . '</span>';
+	><?php
```
Comment (tr1b0t) [inline src/views/v2/list/nav/next.php]: :no_entry_sign: Opening PHP tag must be on a line by itself

<sub>hash: de1cc5a9a6871689a93cbe3e1f680894</sub>

File: src/views/v2/list/nav/next.php [hunk 5]
```diff
@@ -21,14 +21,12 @@
...
+			$events_label = '<span class="tribe-events-c-nav__next-label-plural">' . tribe_get_event_label_plural() . '</span>';
 			echo wp_kses(
 				/* translators: %s: Event (plural or singular). */
 				sprintf( __( 'Next %1$s', 'the-events-calendar' ), $events_label ),
 				[ 'span' => [ 'class' => [] ] ]
 			);
-		?>
-	</a>
+	?></a>
```
Comment (tr1b0t) [inline src/views/v2/list/nav/next.php]: * :no_entry_sign: Line indented incorrectly; expected 3 tabs, found 1
* :no_entry_sign: Closing PHP tag must be on a line by itself

<sub>hash: 10648b1ebe7bdb97729aa367fb2cbe90</sub>

File: src/views/v2/list/nav/prev-disabled.php [hunk 6]
```diff
@@ -9,19 +9,20 @@
...
-				/* translators: %s: Event (plural or singular). */
-				sprintf( __( 'Previous %1$s', 'the-events-calendar' ), $events_label ),
-				[ 'span' => [ 'class' => [] ] ]
-			);
-		?>
-	</button>
+	<button 
+		class="tribe-events-c-nav__prev tribe-common-b2 tribe-common-b1--min-medium" 
+		disabled
+	><?php
```
Comment (tr1b0t) [inline src/views/v2/list/nav/prev-disabled.php]: :no_entry_sign: Opening PHP tag must be on a line by itself

<sub>hash: bd66699949584fafaf54c1fb61c64cac</sub>

File: src/views/v2/list/nav/prev-disabled.php [hunk 7]
```diff
@@ -9,19 +9,20 @@
...
+		$events_label = '<span class="tribe-events-c-nav__prev-label-plural">' . tribe_get_event_label_plural() . '</span>';
+		echo wp_kses(
+			/* translators: %s: Event (plural or singular). */
+			sprintf( __( 'Previous %1$s', 'the-events-calendar' ), $events_label ),
+			[ 'span' => [ 'class' => [] ] ]
```
Comment (tr1b0t) [inline src/views/v2/list/nav/prev-disabled.php]: * :no_entry_sign: Short array syntax is not allowed
* :no_entry_sign: Short array syntax is not allowed
* :no_entry_sign: Short array syntax is not allowed

<sub>hash: b8a774f120b9b202dd195aa69d10e2a6</sub>

File: src/views/v2/list/nav/prev-disabled.php [hunk 8]
```diff
@@ -9,19 +9,20 @@
...
+		);
+	?></button>
```
Comment (tr1b0t) [inline src/views/v2/list/nav/prev-disabled.php]: * :no_entry_sign: Line indented incorrectly; expected 2 tabs, found 1
* :no_entry_sign: Closing PHP tag must be on a line by itself

<sub>hash: 36688eb2f21be19db0f3527d42ea8dac</sub>

File: src/views/v2/list/nav/prev.php [hunk 9]
```diff
@@ -21,14 +21,12 @@
 		rel="prev"
 		class="tribe-events-c-nav__prev tribe-common-b2 tribe-common-b1--min-medium"
 		data-js="tribe-events-view-link"
-	>
-		<?php
-			$events_label = '<span class="tribe-events-c-nav__prev-label-plural"> ' . tribe_get_event_label_plural() . '</span>';
+	><?php
```
Comment (tr1b0t) [inline src/views/v2/list/nav/prev.php]: :no_entry_sign: Opening PHP tag must be on a line by itself

<sub>hash: 6ffcda5d57682070552f069f03fa244f</sub>

File: src/views/v2/list/nav/prev.php [hunk 10]
```diff
@@ -21,14 +21,12 @@
...
+			$events_label = '<span class="tribe-events-c-nav__prev-label-plural">' . tribe_get_event_label_plural() . '</span>';
 			echo wp_kses(
 				/* translators: %s: Event (plural or singular). */
 				sprintf( __( 'Previous %1$s', 'the-events-calendar' ), $events_label ),
 				[ 'span' => [ 'class' => [] ] ]
 			);
-		?>
-	</a>
+	?></a>
```
Comment (tr1b0t) [inline src/views/v2/list/nav/prev.php]: * :no_entry_sign: Line indented incorrectly; expected 3 tabs, found 1
* :no_entry_sign: Closing PHP tag must be on a line by itself

<sub>hash: 98f98ff6a2ccdbecc25e32a75855ad14</sub>

File: src/views/v2/list/nav/next-disabled.php (hunk 2 above)
Comment (bordoni) [reply src/views/v2/list/nav/next-disabled.php]: Ignore this.

File: src/views/v2/list/nav/prev-disabled.php (hunk 7 above)
Comment (bordoni) [reply src/views/v2/list/nav/prev-disabled.php]: Ignore.

**Pull request closed without merging**